### PR TITLE
Update + docs to allow integration tests to pass without requiring credentials/API calls, via replays.

### PR DESCRIPTION
**Pull request by kfwls**

### CODE REVIEW

**Comment by cbetta** [comment]
@kfwls  thanks! Will check this later. Can you have a look and see why the checks fail on the CI server?

**Comment by kfwls** [comment]
Am fixing now - CI needs a newer version of nuget to get one of the packages.

**Comment by kfwls** [comment]
Resolved.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ This library targets .NET 4.5.2 and can be installed via Nuget :
 Install-Package InterFAX.Api -Version 1.0.5
 ```
 
+.NET 3.5/4.0 solutions must make calls directly to the REST API [(documentation)](https://www.interfax.net/en/dev/rest/reference) [(sample code)](https://github.com/kfwls/interfax-dotnet).
```
Comment (cbetta) [inline README.md]: @kfwls thanks for adding this, but we've put this in the samples repo. Could you update it? Then I'm happy to merge.

https://github.com/interfax/interfax-dotnet-samples/blob/master/extra_5_send_individual_fax_dotnet_35.cs

Comment (kfwls) [inline README.md]: Done ref https://github.com/interfax/interfax-dotnet/pull/14/commits/94ffd1c26417395abed64538440d406de8b1e748

**Pull request merged**